### PR TITLE
Onboard object storage credentials-group

### PR DESCRIPTION
**Pull request by vicentepinto98**

### CODE REVIEW

File: internal/cmd/object-storage/credentials-group/delete/delete.go [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+				return err
+			}
+
+			credentialsGroupLabel, err := objectStorageUtils.GetCredentialsGroupName(ctx, apiClient, model.ProjectId, model.CredentialsGroupId)
+			if err != nil {
+				credentialsGroupLabel = model.CredentialsGroupId
+			}
+
+			if !model.AssumeYes {
+				prompt := fmt.Sprintf("Are you sure you want to delete credentials group %s? (This cannot be undone)", credentialsGroupLabel)
```
Comment (joaopalet) [inline internal/cmd/object-storage/credentials-group/delete/delete.go:59]:
```suggestion
				prompt := fmt.Sprintf("Are you sure you want to delete credentials group %q? (This cannot be undone)", credentialsGroupLabel)
```

File: internal/cmd/object-storage/credentials-group/delete/delete.go [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+			}
+
+			// Call API
+			req := buildRequest(ctx, model, apiClient)
+			_, err = req.Execute()
+			if err != nil {
+				return fmt.Errorf("delete Object Storage credentials group: %w", err)
+			}
+
+			cmd.Printf("Deleted credentials group %s\n", credentialsGroupLabel)
```
Comment (joaopalet) [inline internal/cmd/object-storage/credentials-group/delete/delete.go:73]:
```suggestion
			cmd.Printf("Deleted credentials group %q\n", credentialsGroupLabel)
```

File: internal/cmd/object-storage/credentials-group/list/list.go [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+		Short: "Lists all credentials groups",
+		Long:  "Lists all credentials groups.",
+		Args:  args.NoArgs,
+		Example: examples.Build(
+			examples.NewExample(
+				`List all credentials groups`,
+				"$ stackit object-storage credentials-group list"),
+			examples.NewExample(
+				`List all credentials groups in JSON format`,
+				"$ stackit object-storage credentials list --output-format json"),
```
Comment (joaopalet) [inline internal/cmd/object-storage/credentials-group/list/list.go:41]:
```suggestion
				"$ stackit object-storage credentials-group list --output-format json"),
```

File: internal/cmd/object-storage/credentials-group/list/list.go [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+			examples.NewExample(
+				`List up to 10 credentials groups`,
+				"$ stackit object-storage credentials list --limit 10"),
```
Comment (joaopalet) [inline internal/cmd/object-storage/credentials-group/list/list.go:44]:
```suggestion
				"$ stackit object-storage credentials-group list --limit 10"),
```

File: internal/pkg/services/object-storage/utils/utils_test.go [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+			listCredentialsGroupsResp: &objectstorage.ListCredentialsGroupsResponse{
+				CredentialsGroups: &[]objectstorage.CredentialsGroup{
+					{
+						CredentialsGroupId: utils.Ptr(testCredentialsGroupId),
+						DisplayName:        nil,
+					},
+				},
+			},
+			isValid: false,
+		},
```
Comment (joaopalet) [inline internal/pkg/services/object-storage/utils/utils_test.go:106]: Maybe one more test case for the `DisplayName == ""` case?

Comment (GokceGK) [reply internal/pkg/services/object-storage/utils/utils_test.go:106]: Unit test has been added

**Review by joaopalet — APPROVED** [review]

**Pull request merged**